extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/CalciteSelectQueryMSQTest.java

```diff
@@ -161,4 +165,19 @@ public void testQueryWithMoreThanMaxNumericInFilter()
 
   }
 
+  @Test
+  @Override
+  public void testArrayAggQueryOnComplexDatatypes()
```

Comment:
Would be worth including an explanation that this feature does not work on MSQ, but does work with "classic" queries. Might be wroth also adding links between the tests to cross-reference the MSQ vs. "classic" case.

That way, if this ever changes in the future, folks can sort out where the two copies of the test live.

Reply:
Done. Added comments for both tests